spec/models/user_spec.rb

```diff
@@ -65,13 +28,7 @@
 
   describe '#role?' do
     it 'checks if a role is included' do
-      Role.create({ name: "Admin" })
-      user = User.create!({
-                             name: 'Alex',
-                             password: 'testpassword',
-                             email: 'fake@fake.com',
-                             role_ids: [1]
-                           })
+      user = User.create!(attributes_for(:user, name: 'Adam', role_ids: [1]))
```

Comment:
The create syntax (like you used [here](https://github.com/HParker/horizon/pull/25/files#diff-12b107c16792b9ecba685e51b51826f1R17)) is preferred over calling create with attributes for.